src/natcap/invest/carbon.py

```diff
@@ -94,16 +94,16 @@
                 "A table that maps the land-cover IDs to carbon pools.  "

```

Comment:
3 different spellings currently used: "landcover", "land cover", or "land-cover", should all be "land cover" IMO.
```suggestion
                "A table that maps each land cover class to its respective carbon pools.  "

```

Reply:
this would be a great thing to have a style guide for!

Reply:
And also for referring to each land cover class as such and not as an ID ... I bet there's some variation in terminology there as well!

Big 👍 for consistency and building out a style guide for our documentation!